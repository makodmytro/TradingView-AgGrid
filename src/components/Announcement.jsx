import React from 'react';
import styles from './Announcements.module.scss';

export const Announcement = ({ title, children }) => {
    return (
        <div className={styles.announcement}>
            {title && <h4>{title}</h4>}

            {children}
        </div>
    );
};
