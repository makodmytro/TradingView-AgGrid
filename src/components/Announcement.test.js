import React from 'react';
import renderer from 'react-test-renderer';
import { Announcement } from './Announcement';

describe('Announcement', () => {
    it('renders with title', () => {
        const tree = renderer
            .create(<Announcement title="My Title">My content</Announcement>)
            .toJSON();

        expect(tree).toMatchSnapshot();
    });
});