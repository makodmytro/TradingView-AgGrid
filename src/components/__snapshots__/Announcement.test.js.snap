// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Announcement renders with title 1`] = `
<div
  className="announcement"
>
  <h4>
    My Title
  </h4>
  My content
</div>
`;
